README.md

```diff
@@ -133,16 +133,16 @@ Don't see your favorite package manager? Visit our [package managers page](https
 ### Linux/FreeBSD
 
 <!-- x-release-please-start-version -->
-- **AppImage**: [ipfs-desktop-0.31.0-linux-x86_64.AppImage](https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.31.0/ipfs-desktop-0.31.0-linux-x86_64.AppImage)\
-[![](https://img.shields.io/github/downloads/ipfs-shipyard/ipfs-desktop/v0.31.0/ipfs-desktop-0.31.0-linux-x86_64.AppImage.svg?style=flat-square&label=downloads)](https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.31.0/ipfs-desktop-0.31.0-linux-x86_64.AppImage)
-- **Tarball** (use this for building packages for distros): [ipfs-desktop-0.31.0-linux-x64.tar.xz](https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.31.0/ipfs-desktop-0.31.0-linux-x64.tar.xz)\
-[![](https://img.shields.io/github/downloads/ipfs-shipyard/ipfs-desktop/v0.31.0/ipfs-desktop-0.31.0-linux-x64.tar.xz.svg?style=flat-square&label=downloads)](https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.31.0/ipfs-desktop-0.31.0-linux-x64.tar.xz)
-- **Debian** (experimental): [ipfs-desktop-0.31.0-linux-amd64.deb](https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.31.0/ipfs-desktop-0.31.0-linux-amd64.deb)\
```

Comment:
these links are not being updated correctly, but the custom plugin doesn't seem to be working and the `extra-files` and https://github.com/googleapis/release-please/blob/main/src/updaters/generic.ts#L103 still seem to be grabbing these. because of the `<!-- x-release-please-start-version -->` tags.